terminus-op/values.yaml

```diff
@@ -0,0 +1,47 @@
+replicaCount: 1
+
+statefulset:
+  port: 6363
+  image:
+    name: terminusdb/terminusdb-server
+    pullPolicy: Always
```

Comment:
Do we need always?

Reply:
Yes! Docker Images are weird and updating them is a must!